### PR TITLE
SNOW-532795: Bumped up PythonConnector PATCH version from 2.7.2 to 2.…

### PR DESCRIPTION
**Pull request by sfc-gh-kdama**
@noreview - This is an automated process. No review is required

Co-authored-by: Jenkins User <snowflake-provisioner>

Please answer these questions before submitting your pull requests. Thanks!

1. What GitHub issue is this PR addressing? Make sure that there is an accompanying issue to your PR.

   Fixes #NNNN

2. Fill out the following pre-review checklist:

   - [ ] I am adding a new automated test(s) to verify correctness of my new code
   - [ ] I am adding new logging messages
   - [ ] I am modifying authorization mechanisms
   - [ ] I am adding new credentials
   - [ ] I am modifying OCSP code
   - [ ] I am adding a new dependency

3. Please describe how your code solves the related issue.

   Please write a short description of how your code change solves the related issue.

### CODE REVIEW

File: DESCRIPTION.md [hunk 1]
```diff
@@ -9,6 +9,20 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
...
+
+   - shifted package metadata from setup.py to setup.cfg
+   - Add Timezone to Timestamp_TZ 
+   - Fix Storage Credential Error
+   - fix py.typed not being included in wheels
+   - Negative Numbers Mangled for arrow_number_to_decimal
+   - Add GET Empty File Handling
+   - fix long description rendering for PyPi
+   - handling no sms case for DUO
+   - Update requirements files Update numpy requirement from <1.22.0 to <1.23.0
```
Comment (sfc-gh-kwagner) [inline DESCRIPTION.md:23]: Can we add this line:

"Add the ability to auto-create a table when writing a pandas DataFrame to a Snowflake table"

**Review by sfc-gh-kwagner — APPROVED** [review]
LGTM other than minor comment

File: DESCRIPTION.md [hunk 2]
```diff
@@ -9,6 +9,21 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 # Release Notes
 
 
+
+- v2.7.3(January 22,2022)
+
+   - shifted package metadata from setup.py to setup.cfg
```
Comment (sfc-gh-mkeller) [inline DESCRIPTION.md:15]:
```suggestion
```

File: DESCRIPTION.md [hunk 3]
```diff
@@ -9,6 +9,21 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
...
+   - Add Timezone to Timestamp_TZ
```
Comment (sfc-gh-mkeller) [inline DESCRIPTION.md:16]:
```suggestion
   - Fixed a bug where timezone was missing from retrieved Timestamp_TZ columns
```

File: DESCRIPTION.md [hunk 4]
```diff
@@ -9,6 +9,21 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 # Release Notes
 
 
+
+- v2.7.3(January 22,2022)
+
+   - shifted package metadata from setup.py to setup.cfg
+   - Add Timezone to Timestamp_TZ 
+   - Fix Storage Credential Error
```
Comment (sfc-gh-mkeller) [inline DESCRIPTION.md:17]:
```suggestion
   - Fixed a bug where a long running PUT/GET command could hit a Storage Credential Error while renewing credentials
```

File: DESCRIPTION.md [hunk 5]
```diff
@@ -9,6 +9,21 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
...
+   - fix py.typed not being included in wheels
```
Comment (sfc-gh-mkeller) [inline DESCRIPTION.md:18]:
```suggestion
   - Fixed a bug where py.typed was not being included in our release wheels
```

File: DESCRIPTION.md [hunk 6]
```diff
@@ -9,6 +9,21 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
...
+   - Negative Numbers Mangled for arrow_number_to_decimal
+   - Add GET Empty File Handling
+   - fix long description rendering for PyPi
+   - handling no sms case for DUO
+   - Add the ability to auto-create a table when writing a pandas DataFrame to a Snowflake table
+   - Update requirements files Update numpy requirement from <1.22.0 to <1.23.0
+   - Update CODEOWNERS
```
Comment (sfc-gh-mkeller) [inline DESCRIPTION.md:25]:
```suggestion
```

File: DESCRIPTION.md [hunk 7]
```diff
@@ -9,6 +9,21 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
...
+
+- v2.7.3(January 22,2022)
+
+   - shifted package metadata from setup.py to setup.cfg
+   - Add Timezone to Timestamp_TZ 
+   - Fix Storage Credential Error
+   - fix py.typed not being included in wheels
+   - Negative Numbers Mangled for arrow_number_to_decimal
+   - Add GET Empty File Handling
+   - fix long description rendering for PyPi
```
Comment (sfc-gh-mkeller) [inline DESCRIPTION.md:21]:
```suggestion
   - Fixed rendering of our long description for PyPi
```

File: DESCRIPTION.md [hunk 8]
```diff
@@ -9,6 +9,21 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
...
+   - handling no sms case for DUO
```
Comment (sfc-gh-mkeller) [inline DESCRIPTION.md:22]:
```suggestion
   - Fixed a bug where DUO authentication ran into errors if sms authentication was disabled for the user
```

File: DESCRIPTION.md [hunk 9]
```diff
@@ -9,6 +9,21 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
...
+   - Add the ability to auto-create a table when writing a pandas DataFrame to a Snowflake table
+   - Update requirements files Update numpy requirement from <1.22.0 to <1.23.0
```
Comment (sfc-gh-mkeller) [inline DESCRIPTION.md:24]:
```suggestion
   - Bumped the maximum dependency version of numpy from <1.22.0 to <1.23.0
```

File: DESCRIPTION.md [hunk 10]
```diff
@@ -9,6 +9,21 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
...
 
+
+- v2.7.3(January 22,2022)
+
+   - shifted package metadata from setup.py to setup.cfg
+   - Add Timezone to Timestamp_TZ 
+   - Fix Storage Credential Error
+   - fix py.typed not being included in wheels
+   - Negative Numbers Mangled for arrow_number_to_decimal
+   - Add GET Empty File Handling
```
Comment (sfc-gh-mkeller) [inline DESCRIPTION.md:20]:
```suggestion
   - Improved the error message that is encountered when running GET for a non-existing file
```

File: DESCRIPTION.md [hunk 11]
```diff
@@ -9,6 +9,21 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
...
 
 
+
+- v2.7.3(January 22,2022)
+
+   - shifted package metadata from setup.py to setup.cfg
+   - Add Timezone to Timestamp_TZ 
+   - Fix Storage Credential Error
+   - fix py.typed not being included in wheels
+   - Negative Numbers Mangled for arrow_number_to_decimal
```
Comment (sfc-gh-mkeller) [inline DESCRIPTION.md:19]:
```suggestion
   - Fixed a bug where negative numbers were mangled when fetched with the connection parameter arrow_number_to_decimal
```

**Review by sfc-gh-mkeller — CHANGES_REQUESTED** [review]
The lines that I removed are administrative, or non important for strictly users of the library

**Review by sfc-gh-mkeller — APPROVED** [review]

**Pull request merged**